stdlib/queue.gr

```diff
@@ -13,25 +13,34 @@ export let isEmpty = (queue) => {
     }
 }
 
-export let head = (queue) => {
+export let peek = (queue) => {
     match (queue) {
         { forwards: [], backwards: [] } => None,
         { forwards, backwards } => List.head(forwards)
     }
 }
 
-export let enqueue = (value, queue) => {
+/// @deprecated
```

Comment:
```suggestion
/**
  @deprecated Please use `Queue.peek` instead. `Queue.head` will be removed in 0.4.0.
*/
```
Shall we use docblocks since my WIP graindoc stuff only parses them (and will be JSDoc-style aware)

Reply:
Sounds great!